lib/commands/implementations/live.js

```diff
@@ -0,0 +1,15 @@
+const moment = require('moment');
+const Command = require('../command-interface');
+const CommandOutput = require('../command-output');
+
+async function getLiveStatus(input, services) {
+  const { isLive, lastStartTime } = await services.twitch.getStreamStatus();
```

Comment:
I wouldn't mind spending the time converting the entire code base to async/await tbh.

Reply:
I actually don't like async/await. 

I'd rather not give my whole schpeel, but it's essentially syntactic sugar over promises, and I feel like in most cases, it makes error handling more cumbersome than then() catch() chains, as well as harder to reason about where a promises errors are being handled.

It's just syntactic sugar, I feel like the payoffs aren't worth the little bit of abstraction you get for doing it except in very specific cases.

If it wasn't syntactic sugar over promsie's and had its own idiomatic way of error handling I'd be all for it, but it's just not there for me yet.

Reply:
> but it's essentially syntactic sugar over promises

Yeah, it's essentially a feature lifted from C# where async/await is purely syntactic sugar over the TPL

>as well as harder to reason about where a promises errors are being handled.

How so? Using async/await would make error handling no different than if doing synchronous code, no?

>If it wasn't syntactic sugar over promsie's and had its own idiomatic way of error handling I'd be all for it, but it's just not there for me yet.

I doubt this would change, the whole point of async/await is to have the code behave as if it was synchronous.